technical_architecture/performances_guide/batch_page_size.rst

```diff
@@ -0,0 +1,13 @@
+Job product batch size
+======================
+
+A product catalog is unique to one's need. It is configured with a custom number of attributes, families, locales, channels, etc. Those different combinations may affect the import/exports performances.
+
+Indeed the size of the products will define how much RAM they will take to be processed. During product import/exports, we regularly flush the processed products into the database to avoid too much memory consumption.
+
+But, if we flush them too often, the transaction time can slow the process: flushing is not free and doing it too often will be inefficient. On the other hand, if we flush too sporadically, the RAM consumption can increase to a point where the PHP's garbage collector is called too many times and slows down the import/exports.
+
+That's why there is no universal answer to the `pim_job_product_batch_size` parameter and if you encounter performance issues on your import/exports, you may need to tweak it.
```

Comment:
api is import and export as well but we don't have any batch size.
`Flat import file`, `jobs`,  check it with PO

Reply:
Yes, please

Reply:
To me API is enrichment but you can talk about job profiles because it is also true for bulk actions like mass edit and publish products